client/src/components/CategoryNav.tsx

```diff
@@ -27,6 +35,7 @@ function CategoryNav({ onCategoryChange, categories }: CategoryNavProps) {
       </TabsList>
     </Tabs>
   );
-};
+}
 
 export default CategoryNav;
+export default CategoryNav;
```

Comment:
Another merge whoopsy
```suggestion
export default CategoryNav;
```